### PR TITLE
Misc minor

### PR DESCRIPTION
**Pull request by fqqb**

### CODE REVIEW

File: yamcs-core/src/main/java/org/yamcs/archive/CcsdsTmIndex.java
Comment (xpromache) [inline yamcs-core/src/main/java/org/yamcs/archive/CcsdsTmIndex.java:1]: I think you can put long time = getTime(tuple) after if packet.length<7{}

**Review by xpromache — COMMENTED** [review]
You make the packet binary optional adding its size instead but I don't see where you actually have an option to skip the binary from being sent.
Hmm this comment was meant for the "Speed up packets page" commit

**Comment by fqqb** [comment]
It's left out of the response using response filtering (which works on all responses).

https://github.com/yamcs/yamcs/blob/efbc9a0b494d443ea38f6b93ef6b4d590218e380/yamcs-web/src/main/webapp/projects/webapp/src/app/telemetry/packets/PacketsDataSource.ts#L67

(this is post building the proto)

**Comment by xpromache** [comment]
ok, we should check all the ByteString.copyOf and replace them with UnsafeByteOperations.unsafeWrap when possible (I suspect most places)

**Pull request merged**